src/grimp/application/usecases.py

```diff
@@ -184,3 +203,21 @@ def _is_external(module: Module, found_packages: Set[FoundPackage]) -> bool:
         module.is_descendant_of(package_module) or module == package_module
         for package_module in package_modules
     )
+
+
+def _create_chunks(module_files: List[ModuleFile], *, n_chunks: int) -> List[List[ModuleFile]]:
```

Comment:
🐼 Always sceptical when I see a list 😉 .

Maybe `tuple[tuple[ModuleFile], ...]` would be a slightly better return type as it's immutable?

Also considered `frozenset[frozenset[ModuleFile]]` but I wonder if hashing might incur a penalty here.